cluster/src/main/java/com/hartwig/pipeline/execution/vm/GoogleComputeEngine.java

```diff
@@ -148,9 +148,6 @@ public PipelineStatus submit(final RuntimeBucket bucket, final VirtualMachineJob
 
                 if (arguments.usePreemptibleVms()) {
                     final Scheduling.Builder schedulingBuilder = Scheduling.newBuilder().setProvisioningModel("SPOT");
-                    if (arguments.useLocalSsds()) {
-                        schedulingBuilder.setInstanceTerminationAction("DELETE");
```

Comment:
What happens in practice here? Will the Java library throw if this isn't set?

If I try to remove a VM with local SSDs via the Console it will display an error. I wonder if you added this to avoid a real problem you encountered?

Reply:
Nothing is thrown actually. It will behave as it did before, the VM actually appears stopped but you can't start it up again.